lib/ui/login/login_screen.dart

```diff
@@ -0,0 +1,84 @@
+
+class LoginScreen extends StatelessWidget {
+  const LoginScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    SystemChrome.setSystemUIOverlayStyle(const SystemUiOverlayStyle(
+        statusBarColor: Colors.transparent,
+        statusBarIconBrightness: Brightness.light,
+        statusBarBrightness: Brightness.dark));
+    return Scaffold(
+        backgroundColor: Colors.transparent,
+        body: Container(
+          decoration: BoxDecoration(
+            image: DecorationImage(
+              image: AssetImage(Assets.images.bgLoginOverlay.path),
+              fit: BoxFit.cover,
+            ),
+          ),
+          child: Padding(
+            padding:
+                const EdgeInsets.symmetric(horizontal: Dimensions.paddingLarge),
+            child: Center(
+              child: Column(
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: [
+                  Padding(
+                      padding: const EdgeInsets.only(
+                          bottom: 109.0, left: 104.0, right: 104.0),
+                      child: Image.asset(
+                        Assets.images.icNimbleWhiteLogo.path,
+                      )),
+                  Padding(
+                      padding: const EdgeInsets.only(
+                          bottom: Dimensions.paddingMedium),
+                      child: InputFieldWidget(
+                        key: const Key('login_email'),
+                        textHint: AppLocalizations.of(context)!.loginEmail,
+                        keyboardType: TextInputType.emailAddress,
+                        textInputAction: TextInputAction.next,
+                      )),
+                  Padding(
+                      padding: const EdgeInsets.only(
+                          bottom: Dimensions.paddingMedium),
+                      child: InputFieldWidget(
+                          key: const Key('login_password'),
+                          textHint: AppLocalizations.of(context)!.loginPassword,
+                          keyboardType: TextInputType.visiblePassword,
+                          isPassword: true,
+                          textSuffixButton:
+                              AppLocalizations.of(context)!.loginForgotPassword,
+                          textSuffixButtonCallback: null)),
+                  ElevatedButton(
+                    onPressed: () {},
+                    style: ElevatedButton.styleFrom(
+                      minimumSize:
+                          const Size.fromHeight(Dimensions.buttonHeight),
+                      backgroundColor: Colors.white,
+                      shape: RoundedRectangleBorder(
+                        borderRadius:
+                            BorderRadius.circular(Dimensions.radiusNormal),
+                      ),
+                    ),
+                    child: Text(
+                      AppLocalizations.of(context)!.login,
```

Comment:
Please use `context.localization` instead.

```suggestion
                      context.localization.login,
```

Reply:
@nmint8m I am not so sure about this usage, would you mind explaining more about this as I used `context.localiztion` as your suggestion, it raised me this error 🤔 

<img width="1000" alt="Screenshot 2023-04-12 at 2 12 22 PM" src="https://user-images.githubusercontent.com/25218255/231380104-21e05374-c63f-4f39-bb64-30e60eabeae5.png">